.gitignore

```diff
@@ -0,0 +1,9 @@
+# Various IDEs configs
+.idea/
+.vscode/
+
```

Comment:
IDE files should not be ignored in the project configuration but in the user global ignore file